plugins/jira/commands/update-qa-contact.md

```
@@ -0,0 +1,1189 @@
+### API Endpoints Used
+
+**Search Users:**
+```
+GET https://redhat.atlassian.net/rest/api/3/user/search
+Parameters:
+  - query: User name or email to search
+  - maxResults: Maximum number of results (default: 5)
+```
+
+**Search Issues:**
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)